.goreleaser.yml

```diff
@@ -1,8 +0,0 @@
-project_name: terraform-plugin-log
-build:
-  skip: true
-milestones:
-  - close: true
```

Comment:
The `terraform-plugin-*` projects are set to automatically close the associated `vX.Y.Z` milestone on release via Goreleaser. Something else should handle this now.

Reply:
That was an accidental deletion, thanks for catching it!